src/app-layout/__tests__/utils.tsx

```diff
@@ -39,9 +40,8 @@ export function renderComponent(jsx: React.ReactElement) {
   const wrapper = createWrapper(container).findAppLayout()!;
 
   const isUsingGridLayout = wrapper.getElement().classList.contains(visualRefreshStyles.layout);
-  const isUsingMobile = !!wrapper.findByClassName(testutilStyles['mobile-bar']);
```

Comment:
Not used anywhere any more. All tests now use this pattern

https://github.com/cloudscape-design/components/blob/dde24620bceb97c2fd3e59aa45b0e95af8402313/src/app-layout/__tests__/common.test.tsx#L83

Reply:
`isUsingGridLayout` is the next one to be removed, but in another PR